tests/ctst/steps/pra.ts

```diff
@@ -161,14 +162,16 @@ async function waitForPhase(
     return false;
 }
 
-Given('a DR installed', { timeout: 360000 }, async function (this: Zenko) {
+const installTimeout = 360000;
+Given('a DR installed', { timeout: installTimeout }, async function (this: Zenko) {
     Identity.useIdentity(IdentityEnum.ACCOUNT, Zenko.sites['source'].accountName);
     const credentials = Identity.getCurrentCredentials();
     await createSecret(this, 'drctl-s3-creds', {
         accessKey: Buffer.from(credentials.accessKeyId).toString('base64'),
         secretAccessKey: Buffer.from(credentials.secretAccessKey).toString('base64'),
     });
-    await installPRA(this);
+    await installPRA(this, undefined, `${installTimeout.toString()}ms`);
```

Comment:
I would suggest that the command timeout is slightly lower than the step timeout as it will likely take maybe 1s more, and it's better that we first see the command doing a timeout, and not the step?